models.yml

```diff
@@ -45,13 +56,18 @@
 #     - You can add JSON Schema properties to the fields like `enum`, `description`,
 #       `items`, `maxLength` and `minimum`
 # Additional properties:
+#     - The property `sql` can be filled manually with an sql-statement, which fills this field. This
+#       field can't be filled with an own vaue and is always defined in a view. You can use it on all
+#        field types, not only relations. 
```

Comment:
Too many "fills" and "filled"... Maybe something like this is clearer:
```suggestion
#     - The optional property `sql` may contain an SQL statement which is used to calculate the value of this field. This
#       field does not exist with its own value in a table, but is always defined in a view. It can be used on all
#        field types, not only relations. 
```

Reply:
done